apps/rush-lib/src/cli/actions/RushCommandLineParser.ts

```diff
@@ -117,12 +117,12 @@ export class RushCommandLineParser extends CommandLineParser {
 
       this.addAction(new ChangeAction(this));
       this.addAction(new CheckAction(this));
-      this.addAction(new GenerateAction(this));
```

Comment:
Generate shouldn't be entirely removed. Instead, it should return a nonzero exit code and print that it's deprecated and the user should run `update` instead.

Reply:
It would also be useful to run an "upgrade helper" before ts-command-line, so we can keep the new implementation clean and we can keep the old stuff in one place.

---
In reply to: [187453862](https://github.com/Microsoft/web-build-tools/pull/646#discussion_r187453862) [](ancestors = 187453862)

Reply:
Good idea -- implemented

---
In reply to: [187454181](https://github.com/Microsoft/web-build-tools/pull/646#discussion_r187454181) [](ancestors = 187454181,187453862)